common/test/acceptance/tests/studio/test_studio_grading.py

```diff
@@ -0,0 +1,96 @@
+"""
+Acceptance tests for grade settings in Studio.
+"""
+from common.test.acceptance.pages.studio.settings_graders import GradingPage
+from common.test.acceptance.tests.studio.base_studio_test import StudioCourseTest
+from common.test.acceptance.fixtures.course import XBlockFixtureDesc
+
+
+class GradingPageTest(StudioCourseTest):
+    """
+    Bockchoy tests to add/edit grade settings in studio.
+    """
+
+    url = None
+
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(GradingPageTest, self).setUp()
+        self.grading_page = GradingPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        self.grading_page.visit()
+
+    def populate_course_fixture(self, course_fixture):
+        """
+        Return a test course fixture.
+        """
+        course_fixture.add_children(
+            XBlockFixtureDesc("chapter", "Test Section").add_children(
+                XBlockFixtureDesc("sequential", "Test Subsection").add_children(
+                )
+            )
+        )
+
+    def test_add_grade_range(self):
+        """
+        Scenario: Users can add grading ranges
+            Given I have opened a new course in Studio
+            And I am viewing the grading settings
+            When I add "1" new grade
+            Then I see I now have "3"
+        """
+        for _ in range(1):
```

Comment:
This loop looks redundant; is there a reason not to just execute the content once directly?